packages/offix-client/src/config/OffixClientConfig.ts

```diff
@@ -7,6 +7,11 @@ import { ObjectState } from "offix-offline";
 import { ConflictListener } from "offix-offline";
 import { CacheUpdates } from "offix-cache";
 import { RetryLink } from "apollo-link-retry";
+import { HttpLink } from "apollo-link-http";
+
+export type Omit<T, K extends keyof T> = Pick<T, Exclude<keyof T, K>>;
+
+export type HttpLinkOptions = Omit<HttpLink.Options, "uri">;
```

Comment:
This is just on typescript level. JS side will still accept uri.
I think we just need to pass it but simply force assign url from config.uri after.

Reply:
if you just swap the order of the spread operator you will get this result.